portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -99,7 +122,7 @@ private int getUserChoice(HttpServletRequest request) {
     }
 
     // Check that the input is between 1 and 20.
-    if (userChoice < 1 || userChoice > 20) {
+    if (userChoice < 1 || userChoice > 10) {
```

Comment:
Why should input be between 1 and 10? The comment here should explain why these values are important. Also consider changing 10 to be a static constant value with its own comment.